### ISSUE TITLE
Error in PredictCost() for the op: "Softmax" 

### ISSUE THREAD

**Issue by DetionDX** [akanyaani/gpt-2-tensorflow2.0#21]
I am running this code on my 3090 with tensorflow-gpu 2.5.0 and cuda 11.2

I can successfully run the training, but I get the error as follows:

![Screenshot from 2021-07-08 10-44-53](https://user-images.githubusercontent.com/9487284/124942890-07b37c80-dfda-11eb-979b-a0e59e6a8343.png)

Can someone give me a hint to solve this?
